sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -349,6 +357,17 @@ private[hive] class SparkExecuteStatementOperation(
     }
   }
 
+  def timeoutCancel(): Unit = {
```

Comment:
Could we inline this method in the line 211?

Reply:
i am not sure this method could be inline, because this method should be synchronized on SparkExecuteStatementOperation object
`  def timeoutCancel(): Unit = {
    synchronized {`

Reply:
Ah, I see. btw, `def timeoutCancel()` -> `private def timeoutCancel()`?

Reply:
> Ah, I see. btw, `def timeoutCancel()` -> `private def timeoutCancel()`?
Fixed, thanks!

Reply:
> Ah, I see. btw, `def timeoutCancel()` -> `private def timeoutCancel()`?

Fixed, thanks!